stubs/Php80.phpstub

```diff
@@ -76,11 +174,20 @@ class Attribute
     }
 }
 
-class ReflectionUnionType extends ReflectionType {
+class ReflectionProperty implements Reflector
+{
     /**
-     * @return non-empty-list<ReflectionNamedType>
+     * @return non-empty-string|false
+     *
+     * @psalm-pure
      */
-    public function getTypes() {}
+    public function getDocComment(): string|false {}
+}
+
+/** @psalm-immutable */
+class ReflectionUnionType extends ReflectionType {
+    /** @return non-empty-list<ReflectionNamedType> */
```

Comment:
An unsealed tuple may allow specifying that two first elements are `ReflectionNamedType`s, but it would allow passing other types with offsets 2+. Psalm currently doesn't support array intersections. If it did, it would be possible with `array{ReflectionNamedType,ReflectionNamedType} & list<ReflectionNamedType>`

Reply:
Was thinking the same, but then a tuple is too restrictive anyway

Reply:
Skipping on this, for now: maybe one day :)